navigation/navigation-common/build.gradle

```diff
@@ -53,15 +53,14 @@ androidXMultiplatform {
     sourceSets {
         commonMain {
             dependencies {
-                api "org.jetbrains.compose.annotation-internal:annotation:1.7.1"
-                api "org.jetbrains.compose.collection-internal:collection:1.7.1"
-                api project(":core:core-bundle")
```

Comment:
Are we going to deprecate/remove this package later?

Reply:
yes. we don't need it anymore

Reply:
I deleted core-bundle.

Reply:
It requires a mention in release notes (unlike regular version update that we write in dependencies section) and CI adoption.

cc @igordmn for second approval for such changes

Reply:
I am not sure about a release note here. it was the internal module.

Reply:
I know it was used for some libraries directly, so... we should be really careful here